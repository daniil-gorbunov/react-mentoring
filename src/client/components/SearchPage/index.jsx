import React from 'react';
// import { Row, Col } from 'react-flexbox-grid';
// import Header from '../Header';
// import SearchResults from '../SearchResults';
// import Footer from '../Footer';

const SearchPage = () => (
  <span>Hello cruel world</span>
);

export default SearchPage;
