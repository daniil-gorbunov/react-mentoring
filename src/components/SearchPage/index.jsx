import React from 'react';
import Header from '../Header';
import SearchResultsContainer from '../SearchResultsContainer';
import Footer from '../Footer';

const SearchPage = () => (
  <div>
    <Header />
    <SearchResultsContainer />
    <Footer />
  </div>
);

export default SearchPage;
