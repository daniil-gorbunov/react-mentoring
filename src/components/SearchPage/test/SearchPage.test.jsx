import React from 'react';
import renderer from 'react-test-renderer';
import SearchPage from '..';

jest.mock('../../Header', () => 'Header');
jest.mock('../../SearchResults', () => 'SearchResults');
jest.mock('../../Footer', () => 'Footer');

describe('SearchPage', () => {
  test('renders correctly', () => {
    const searchPage = renderer.create(<SearchPage />);
    expect(searchPage).toMatchSnapshot();
  });
});
