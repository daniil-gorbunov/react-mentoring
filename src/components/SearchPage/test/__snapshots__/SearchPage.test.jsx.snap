// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPage renders correctly 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-xs-12"
    >
      <Header />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-xs-12"
    >
      <SearchResults />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-xs-12"
    >
      <Footer />
    </div>
  </div>
</div>
`;
